$MirOS: ports/net/openvpn/patches/patch-configure_ac,v 1.1.7.1 2005/03/18 15:48:42 tg Exp $
--- configure.ac.orig	Thu Nov  3 06:39:31 2005
+++ configure.ac	Fri Nov  4 20:32:13 2005
@@ -216,8 +216,8 @@ case "$target" in
 *solaris*)
 	AC_DEFINE(TARGET_SOLARIS, 1, [Are we running on Solaris?])
 	;;
-*openbsd*)
-	AC_DEFINE(TARGET_OPENBSD, 1, [Are we running on OpenBSD?])
+*openbsd*|*mirbsd*|*ekkobsd*)
+	AC_DEFINE(TARGET_OPENBSD, 1, [Are we running on OpenBSD or a fork?])
 	;;
 *freebsd*)
 	AC_DEFINE(TARGET_FREEBSD, 1, [Are we running on FreeBSD?])
@@ -408,8 +408,7 @@ if test "$PTHREAD" = "yes"; then
   ACX_PTHREAD(
         [
 	    case "$target" in
-	    *openbsd*)
-		AC_MSG_RESULT([WARNING: pthread support on OpenBSD is unstable!])
+	    *openbsd*|*mirbsd*|*ekkobsd*)
 		CFLAGS="$CFLAGS -pthread"
 		;;
 	    esac
